src/lib/apexPmd.ts

```diff
@@ -169,8 +170,12 @@ export class ApexPmd {
       pmdCmd.addListener('exit', (e) => {
         if (e !== 0 && e !== 4) {
           this.outputChannel.appendLine(`Failed Exit Code: ${e}`);
+          if (!showStdErr) this.outputChannel.appendLine('stderr:' + stderr);
```

Comment:
@adangel I wonder if we should just get rid of this configuration point...  

I think I originally added it because PMD stderr is pretty noisy... 

But really the only reason anyone is opening up the Output panel is to debug problems, and I think potentially having it turned off is causing extra maintainer effort.

Thoughts?

Reply:
As it stands, a user might get the "Check the plugin output for details." message, but not actually see the details in the output panel because `showStdErr` is unchecked.

Reply:
@adangel Any thoughts?  I can go ahead and make the update so we can close this one out.

Reply:
I'll remove these options and update this PR